blogs_notebooks/titanic_data_science_solutions_blog_1.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>RAPIDS is misspelled. We should also introduce the user to the data, such as what it is, what we will be doing to that data, why it may be/what we will do is important, and how and where they can find it (attribute the source).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>